homeassistant/components/knx/services.yaml

```diff
@@ -40,6 +40,17 @@ read:
       example: "1/1/0"
       selector:
         object:
+restart_device:
+  name: "Restart a device on the KNX bus"
+  description: "Send a basic restart request to a device on the KNX bus."
+  fields:
+    address:
+      name: "Individual address"
+      description: "Individual address(es) to send restart request to. Single individual address or list accepted."
+      required: true
+      example: "1.1.3"
+      selector:
+        object:
```

Comment:
Things like these should be represented as a button, not a service.

Reply:
We can't know the devices address so we need that input.

Reply:
So a user needs the possibility to create button entities first?